crates/josh-sync/src/sync.rs

```diff
@@ -77,6 +80,7 @@ impl GitSync {
             "--depth=1",
         ]);
         let new_summary = check_output(["git", "log", "-1", "--format=%h %s", &new_upstream_base]);
+        let new_summary = replace_references(&new_summary, &self.upstream_repo);
```

Comment:
If this interest you, in triagebot bot we are using this regex, https://github.com/rust-lang/triagebot/blob/1657187393c4ed9979dfab3901a67b4b832cade0/src/handlers/issue_links.rs#L18-L19

Reply:
To avoid false positives (like `#123az`) I would also highly recommend using `\b` at the end.

Reply:
Indeed, missed that. Added, thanks